modules/create-wif-cluster-cli.adoc

```diff
@@ -0,0 +1,192 @@
+// Module included in the following assemblies:
+//
+// * osd_install_access_delete_cluster/creating-a-gcp-cluster-with-workload-identity-federation.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="create-wif-cluster-cli_{context}"]
+= Creating a Workload Identity Federation cluster using the OCM CLI
+
+You can create an {product-title} on {GCP} cluster with Workload Identity Federation (WIF) using the OpenShift Cluster Manager CLI (`ocm`) in interactive or non-interactive mode.
+
+Before creating the cluster, you must first create a WIF configuration. You can create a WIF configuration in auto or manual mode.
```

Comment:
```suggestion
Before creating the cluster, you must first create a WIF configuration. 
```
You talk about manual and auto mode in the procedure so I don't think you need to mention it here too.